index.html

```diff
@@ -254,7 +254,7 @@ <h3>Resources Included in the <a>PerformanceResourceTiming</a>
 <a data-cite=
 "PERFORMANCE-TIMELINE-2#performance-timeline">Performance
 Timeline</a>, unless excluded from the timeline as part of the <a
-href="#processing-model">processing model</a>.
+href="FETCH#concept-fetch">fetching process</a>.
```

Comment:
Are all exclusions happening in Fetch? e.g. where would we be dropping exposure of cross-origin no-cors fetched-by-CSS resources?

Reply:
Yes, all exclusions happen in FETCH. The `timing info` is attached to a response only if we're passed the CORS checks, and at that point it's also cleaned up for TAO.